backend/corpora/common/entities/entity.py

```diff
@@ -49,6 +53,45 @@ def list(cls) -> "Entity":
         """
         return [cls(obj) for obj in cls.db.query([cls.table])]
 
+    @classmethod
+    def list_attributes_in_time_range(
```

Comment:
moved from project.py

Reply:
Why is it moved from `project.py`? What's the use case for the other entity types?

Reply:
True, I didn't end up using in a more generalized way. I'll return it.